cargo-dist/src/tasks.rs

```diff
@@ -2112,6 +2120,8 @@ impl<'pkg_graph> DistGraphBuilder<'pkg_graph> {
             warn!("The Homebrew publish job is enabled but no tap was specified\n  consider setting the tap field in Cargo.toml");
         }
 
+        let runtime_conditions = pick_runtime_conditions(&self.manifest.systems);
```

Comment:
This can now be replaced with 

```
release.platform_support.conflated_runtime_conditions()
```

And stuff like glibc_thunderdome I think are (alas) obsoleted?